### PR TITLE
Update core to 9.2.13. SA-CORE-2022-003

### PR DESCRIPTION
**Pull request by rasben**
- composer update drupal/core "drupal/core-*" --with-all-dependencies
- drush updb -y
- drush cex -y

### CODE REVIEW

**Comment by rasben** [comment]
I'm locking it specifically to 9.2.* for now, as I'm not confident enough with this site to do a jump straight to 9.3.*

Obviously, we need to update to 9.3 soon, but not now :)

**Comment by rasben** [comment]
Visual "errors" are false alarms.

**Comment by arnested** [comment]
backstop-check approve

File: composer.json [hunk 1]
```diff
@@ -67,9 +67,9 @@
         "drupal/chosen_lib": "^2.6.0",
         "drupal/cookiebot": "1.0.0-alpha8",
         "drupal/cookieconsent": "^1.6",
-        "drupal/core-composer-scaffold": "^9",
-        "drupal/core-project-message": "^9",
-        "drupal/core-recommended": "^9",
+        "drupal/core-composer-scaffold": "9.2.*",
+        "drupal/core-project-message": "9.2.*",
+        "drupal/core-recommended": "9.2.*",
```
Comment (arnested) [inline composer.json:72]: You actually allow composer to resolve core into a version below the intended 9.2.13. E.g. 9.2.11.

I think  `~9.2.13` is the way to go.

**Review by arnested — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rasben) [reply composer.json]: FiIiIiiNEeEe

https://github.com/reload/ddsdk2016/compare/fadb657bab0a9da8461987bbc1986816a9bc6436..41c6e685592e9f6d31467b23c92605e14b7287a7

**Comment by arnested** [comment]
backstop-check approve

**Review by arnested — APPROVED** [review]
LGTM :+1:

**Pull request merged**